index-test.js

```diff
@@ -1,4 +1,13 @@
 // This is nasty, but prevents requiring different commands
 // for windows machines
 process.env.DEBUG = '-*';
-require('dotenv').config({ path: `${__dirname}/.env` });
+
+// Hacky check for dev/CI startup
+const fs = require('fs');
+
+try {
+  fs.statSync(`${__dirname}/.env`);
+  require('dotenv').config({ path: `${__dirname}/.env` });
+} catch (e) {
+  console.log('CI Build, reading from environment.');
```

Comment:
This needs to just return instead of being a console log - it gets called multiple times. Travis also needs to get setup with env variables